common/src/main/java/com/hortonworks/streamline/common/SchemaValueConverter.java

```diff
@@ -70,8 +72,9 @@ private SchemaValueConverter() {
                     requiredFieldsNotFoundInValue.stream().map(Schema.Field::getName).collect(toList()));
         }
 
-        return value.entrySet().stream()
-                .collect(toMap(Map.Entry::getKey, e -> convert(fieldToType.get(e.getKey()), e.getValue())));
```

Comment:
`toMap` does not allow null values. Replacing with below to support null values in future.

Reply:
I didn't notice it. Nice finding. 
Looks like related issue here: https://bugs.openjdk.java.net/browse/JDK-8148463